subprojects/platform-native/src/testFixtures/groovy/org/gradle/nativeplatform/fixtures/AvailableToolChains.java

```diff
@@ -285,7 +291,7 @@ public boolean accept(File swiftInstall) {
         CLANG("clang"),
         VISUAL_CPP("visual c++"),
         MINGW_GCC("mingw"),
-        CYGWIN_GCC("gcc cygwin"),
+        CYGWIN_GCC_64("gcc cygwin"),
```

Comment:
❓ Why is the `ToolFamily` for `gcc cygwin` include the architecture? It doesn't seem to make any sense here.